docker/config/config.json

```diff
@@ -0,0 +1,120 @@
+{
```

Comment:
Why do we add a config in a docs PR?

Anyway, this file is missing a newline at it's end

Reply:
Shouldnt be there. probably a left over from testing the setup